frontend/src/app/main-table/volumes-table/volume-table.component.ts

```diff
@@ -57,20 +58,20 @@ export class VolumeTableComponent implements OnChanges {
     const dialogRef = this.dialog.open(ConfirmDialogComponent, {
       width: "fit-content",
       data: {
-        title: "You are about to delete the volume: " + pvc.pvc.name,
-        message:
-          "Are you sure you want to delete this volume? " +
-          "This action can't be undone.",
-        yes: "delete",
-        no: "cancel"
+        title:
+          this.translate.instant("volumeTable.deleteDialogTitle") +
+          pvc.pvc.name,
+        message: this.translate.instant("volumeTable.deleteDialogMessage"),
+        yes: this.translate.instant("volumeTable.deleteDialogYes"),
+        no: this.translate.instant("volumeTable.deleteDialogNo")
       }
     });
 
     dialogRef
       .afterClosed()
       .pipe(first())
       .subscribe(result => {
-        if (result !== "delete") {
+        if (result !== this.translate.instant("volumeTable.deleteDialogYes")) {
```

Comment:
Same as above: lookup once and reference twice. :straight_ruler: :straight_ruler: :hocho: